inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.h

```diff
@@ -55,23 +58,45 @@ class MKLDNNPadNode : public MKLDNNNode {
         size_t workAmount = 0lu;
         size_t lastDstDim = 1lu;
         size_t shift = 0lu;
-        uint8_t sizeData = 1;
-    } params;
-
-    template<typename T>
-    struct PadConstantEmitter {
-        void operator()(MKLDNNPadNode* node) {
-            node->padConstantCommon<T>();
-        }
+        PadMode padMode;
+
+        MKLDNNMemoryPtr srcMemPtr;
```

Comment:
I guess we shouldn't store `MKLDNNMemory` objects because we will share `PadExecutor` between all `MKLDNNPadNode` in model. So memory objects will be different for nodes.
Better to pass them as arguments `exec`

Reply:
Yes, right. Removed, thanks!